src/main/java/dinkplugin/notifiers/QuestNotifier.java

```diff
@@ -28,22 +45,40 @@ public void onWidgetLoaded(WidgetLoaded event) {
         if (event.getGroupId() == QUEST_COMPLETED_GROUP_ID && isEnabled()) {
             Widget quest = client.getWidget(WidgetInfo.QUEST_COMPLETED_NAME_TEXT);
             if (quest != null) {
-                this.handleNotify(quest.getText());
+                String questText = quest.getText();
+                // 1 tick delay to ensure relevant varbits have been processed by the client
+                clientThread.invokeLater(() -> handleNotify(questText));
             }
         }
     }
 
     private void handleNotify(String questText) {
+        int completed = client.getVarbitValue(COMPLETED_ID);
+        int total = client.getVarbitValue(TOTAL_ID);
+        boolean validCount = completed > 0 && total > 0;
+
+        int points = client.getVarpValue(VarPlayer.QUEST_POINTS);
+        int possiblePoints = client.getVarbitValue(QP_TOTAL_ID);
+        boolean validPoints = points > 0 && possiblePoints > 0;
```

Comment:
nit: rename these local variables to match what they're called in QuestNotificationData

`completed` -> `completedQuests`
`total` -> `totalQuests`
and then `validCount` -> `validQuests`

`points` -> `questPoints`
`possiblePoints` -> `totalQuestPoints`